### PR TITLE
Result Page

### PR DESCRIPTION
**Pull request by foolhardy21**
This PR has:
- Result Page.
- Use this link for preview https://deploy-preview-3--eager-wilson-9b76d7.netlify.app/result.html

Feel free to review. Thanks.

### CODE REVIEW

**Review by VLeads — APPROVED** [review]
Looks good to go

File: result.html [hunk 1]
```diff
@@ -11,14 +11,25 @@
 </head>
 <body>
 
-    <header class="ques-header padding-xxlg flex-column flex-center margin-btm-lg">
+    <header class="page-header flex-horizontal-start padding-xxlg">
+        <p class=" c-blue text-bold font-size-xlg text-uppercase letter-space-md">
+            sneakerhead
+        </p>
+    </header>
+    <!-- <header class="ques-header padding-xxlg flex-column flex-center margin-btm-lg">
```
Comment (MayankKumar10) [inline result.html]: If this code not necessary you can delete it

File: style.css [hunk 2]
```diff
@@ -3,16 +3,22 @@
...
     background: var(--c-lightblue);
 }
 
 .ques-wrapper {
     width: 35rem;
+    background: var(--c-white);
+}
+
+.material-icons {
+    font-size: 140%;
```
Comment (MayankKumar10) [inline style.css]: avoid using size greater than 100% it will effect when code incresed

**Review by MayankKumar10 — APPROVED** [review]
Everything else looks good you can merge

File: result.html (hunk 1 above)
Comment (foolhardy21) [reply result.html]: thanks for pointing out @MayankKumar10 . i have made the changes.

File: style.css (hunk 2 above)
Comment (foolhardy21) [reply style.css]: thanks. i have replaced it with rem.

**Pull request merged**